src/Sentry/IJsonSerializable.cs

```diff
@@ -15,17 +16,17 @@ public interface IJsonSerializable
         /// Note: this method is meant only for internal use and is exposed due to a language limitation.
         /// Avoid relying on this method in user code.
         /// </remarks>
-        void WriteTo(Utf8JsonWriter writer);
+        void WriteTo(Utf8JsonWriter writer, IDiagnosticLogger logger);
```

Comment:
Technically a breaking change but note the `remarks` given above.

For that reason I'm not planning to bump a major.

Reply:
That'll break CrontribSentry 😂, glad it's a thing from the past.